apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -72,7 +74,8 @@ export default class LibraryPublisher extends React.Component {
       this.props.libraryDetails.libraryName
     ),
     libraryDescription: this.props.libraryDetails.libraryDescription,
-    selectedFunctions: this.props.libraryDetails.selectedFunctions
+    selectedFunctions: this.props.libraryDetails.selectedFunctions,
+    profaneWords: []
```

Comment:
Should this be `null` then, too, to match the expected value when there are no profane words?